django_large_image/templates/django_large_image/_include/colors.html

```diff
@@ -18,15 +18,90 @@
     @click="controls = ! controls"
   >
     Controls
-    <i class="ri-equalizer-line text-lg font-normal" x-show="!controls"></i>
-    <i class="ri-close-line text-lg font-normal" x-show="controls"></i>
+    <i class="ri-equalizer-line" x-show="!controls"></i>
+    <i class="ri-close-line" x-show="controls"></i>
   </button>
 
-  <div class="absolute bottom-full pb-3 right-0" x-show="controls">
-    <div class="card bg-gray-100 bg-opacity-[.90] border border-gray-300 min-w-max !rounded-lg shadow-lg">
-      <div class="card-body">
-        <div class="grid grid-cols-2 gap-8">
-          <div class="controls flex flex-col gap-6">
+  <div
+    class="absolute bottom-full pb-3 right-0"
+    x-show="controls"
+  >
+    <div class="card bg-slate-100 bg-opacity-[.85] min-w-max !rounded-lg shadow-lg">
+      <div
+        class="card-body !p-0"
+        x-data="{ customizeDisplay: false }"
+      >
+        <div
+          class="grid gap-4"
+          :class="{ 'grid-cols-2' : customizeDisplay }"
+        >
+          <div
+            x-show="customizeDisplay"
+            x-data="{ configure: false }"
+            class="row-span-full w-56"
+          >
+            <div
+              class="bg-secondary flex flex-col h-full text-slate-900 justify-between p-4 text-secondary-content"
+              x-data="{ items: [
+                { frame: 'Frame 1', bands: ['Band 1', 'Band 2', 'Band 3'] },
+                { frame: 'Frame 2', bands: ['Band 1', 'Band 2'] }
+              ]}"
```

Comment:
I've just inputted my own data in an array for the bands and frames, but this will need to get blown up as well. I did it just to be able to display them easier in a for-loop. I assumed this would make it easier when the real logic is put in place.